components/authz-service/engine/opa/opa_internal_test.go

```diff
@@ -549,3 +429,866 @@ func randomPolicies(i int) []map[string]interface{} {
 
 	return newPolicies
 }
+
+// v2 benchmarks
+
+var (
+	allProjects = []string{
+		"(unassigned)",
+		"project1",
+		"project2",
+		"project3",
+		"project4",
+		"project5",
+		"project6",
+	}
+	projectsResponse []string
+)
+
+// At time of benchmarking, this is around half a second for 1000 teams
+func BenchmarkV2FilterAuthorizedPairsRealWorldExample(b *testing.B) {
```

Comment:
this test was already written and we just moved it into the v2 section of the file